operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -148,37 +148,63 @@ func (c *CentralComponentSpec) GetAdminPasswordGenerationDisabled() bool {
 	return pointer.BoolPtrDerefOr(c.AdminPasswordGenerationDisabled, false)
 }
 
-// CentralDBEnabled returns a bool if CentralDBSpec is not nil
+// CentralDBEnabled returns true if central db should be created
 func (c *CentralComponentSpec) CentralDBEnabled() bool {
-	if c == nil {
+	if c == nil || c.DB == nil || c.DB.Enabled == nil {
 		return false
 	}
-	return c.DB != nil
+
+	return *c.DB.Enabled == CentralDBEnabledTrue
 }
 
 // CentralDBSpec defines settings for the "central db" component.
 type CentralDBSpec struct {
+	// Specify whether central-db is enabled, Default configures central to use rocksdb.
+	// If Enabled the operator will provision the Central DB or use an external DB if a connection string is provided.
+	//+kubebuilder:validation:Default=Default
+	//+kubebuilder:default=Default
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=1
+	Enabled *CentralDBEnabled `json:"enabled,omitempty"`
+
 	// Specify a secret that contains the password in the "password" data item.
 	// If omitted, the operator will auto-generate a DB password and store it in the "password" item
 	// in the "central-db-password" secret.
-	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Administrator Password",order=1
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Administrator Password",order=2
 	PasswordSecret *LocalSecretReference `json:"passwordSecret,omitempty"`
 
 	// Specify a connection string that corresponds to an existing database. If set, the operator will not manage Central DB.
 	// When using this option, you must explicitly set a password secret; automatically generating a password will not
 	// be supported.
-	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=2
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=3
 	ConnectionStringOverride *string `json:"connectionString,omitempty"`
 
 	// Configures how Central DB should store its persistent data. You can choose between using a persistent
 	// volume claim (recommended default), and a host path.
-	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=3
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=4
 	Persistence *DBPersistence `json:"persistence,omitempty"`
 
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=99
 	DeploymentSpec `json:",inline"`
 }
 
+// CentralDBEnabled is a type for values of spec.central.db.enabled.
+//+kubebuilder:validation:Enum=Default;Enabled
+type CentralDBEnabled string
+
+const (
+	// CentralDBEnabledDefault configures the central to use rocksdb
+	CentralDBEnabledDefault CentralDBEnabled = "Default"
+	// CentralDBEnabledTrue configures the central to use a PostgreSQL database (Technical Preview)
```

Comment:
There's no such thing as "Technical Preview", there's "Technology Preview". https://access.redhat.com/support/offerings/techpreview/
```suggestion
	// CentralDBEnabledTrue configures the central to use a PostgreSQL database (Technology Preview)
	// TODO(ROX-13557): omit mention of Technology Preview.
```

It would be really nice if you could also find+replace all other Technical Previews with Technology Preview in this file.

Reply:
Done.